airplane.py

```diff
@@ -32,6 +31,18 @@ def reset():
 
     return 'DB HAS BEEN RESET'
 
+# test create new customer
+@app.route('/customer')
+def add_customer():
+    cust_name = request.args.get('name')
+    cust_age = request.args.get('age')
+    cust_email = request.args.get('email')
+    cust_phone = request.args.get('phone')
+
+    airdb.add_customer(cust_name, cust_age, cust_email, cust_phone)
+
+    return 'ADDED NEW CUSTOMER %s' % cust_name
```

Comment:
should return the newly created customer ID here so the customer can use it as their reference login number later.